CHANGES.rst

```diff
@@ -171,7 +171,8 @@ Bug fixes
 
 Other Changes and Additions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
+- ``astropy.coordinates``
+  - Added test_sun_02 to test_funcs.py. 
```

Comment:
This doesn't need to go here; adding a new test (that isn't directly associated with fixing an issue) isn't of interest for the changelog.